app/server/models/credit_transfer.py

```diff
@@ -165,8 +165,8 @@ def resolve_as_completed(self, existing_blockchain_txn=None):
         self.resolved_date = datetime.datetime.utcnow()
         self.transfer_status = TransferStatusEnum.COMPLETE
 
-        self.sender_transfer_account.balance -= self.transfer_amount
-        self.recipient_transfer_account.balance += self.transfer_amount
+        self.sender_transfer_account.increment_balance(-1 * self.transfer_amount)
```

Comment:
Why not have `self.sender_transfer_account.decrement_balance()`?

Reply:
No strong reason - just figured it was an unnecessary method given `increment_balance` does both. Could easily add

Reply:
I feel like it would be cleaner to have `increment_balance()` and `decrement_balance()` functions separately given how core they are. should have simple unit tests as well imo

Reply:
My email response decided to shift itself to the bottom of the PR lol:

> Ok added decrement as well. Guess it makes sense given that eth implements
both add and sub.
Unit tests already exist...